scripts/resource-autogen/allowlist/allowlist.go

```diff
@@ -58,7 +58,6 @@ var (
 		"bigquery_analytics_hub/google_bigquery_analytics_hub_data_exchange",
 		"bigquery_analytics_hub/google_bigquery_analytics_hub_listing",
 		"bigquery_connection/google_bigquery_connection",
-		"bigquery_data_transfer/google_bigquery_data_transfer_config",
```

Comment:
Good catch! Could you update the guide to suggest contributors clean up this list if apply?